src/Tests/Microsoft.DotNet.GenAPI.Tests/CSharpFileBuilderTests.cs

```diff
@@ -154,16 +155,367 @@ public partial struct PublicStruct
                     {
                     }
 
-                    internal readonly record struct ReadonlyRecordStruct : System.IEquatable<ReadonlyRecordStruct>
+                    internal readonly partial struct ReadonlyRecordStruct : System.IEquatable<ReadonlyRecordStruct>
```

Comment:
I assume this is expected, but why are the compiler-generated APIs added in a separate partial type and not included in the same one above?

Reply:
This is what the `record` syntax produces.  For structs we cannot identify them, so we just reproduce the members the compiler produced.  See https://github.com/dotnet/roslyn/issues/69326